### DISCUSSION TITLE
Inline query answers cached, any way to disable cache?

### DISCUSSION THREAD

**Discussion by sudousercom** [eternnoir/pyTelegramBotAPI#2035] [Q&A] [answered]
Hi! I have a function with dice roll, and it works fine with command, but if i trying to use it with inline query, i see execution of generation dice roll on first run, but then it does nothing for some time, just gives me a first generated result, after some time, it is regenerate results again, looks like results get cached, and just used from cache for some time. Can i disable cache, or clear inline answer results? None was found in docs for this case.

**Comment by coder2020official** [comment]
As far as I remember answerInlineQuery has cache or some sort of parameter, check it

**Comment by coder2020official** [comment, marked as answer]
Yes, it is cache_time:
https://core.telegram.org/bots/api#answerinlinequery

Set it to 0

**Reply by sudousercom** [reply]
thank you!

**Reply by sboriskin287** [reply]
For me value 0 of cache_time enforces tg server to cache result in 300 secs (i.e by default). For me works cache_time = 1

**Comment by Makcal** [comment]
For me it cache_time = 0 works only on Android and Web version. Windows/Linux desktops still do cache
